resources/views/brand/header.blade.php

```diff
@@ -0,0 +1,14 @@
+@push('head')
+    <link
+        href="/favicon.ico"
+        id="favicon"
+        rel="icon"
+    >
+@endpush
+
+<p class="h2 n-m font-thin v-center">
+    <span class="m-l d-none d-sm-block">
+        Panel Administrativo
+        <small class="v-top opacity">hoydonamos</small>
```

Comment:
Why just `hoydonamos`?

Reply:
Looks simple and stetic but this is relative, would you like to change it ?